### DISCUSSION TITLE
Felhő alapú megoldás 2.

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#19] [Kérdések és válaszok]
Hogyan tudja a NAV biztosítani és ellenőrzi azt, hogy a felhőszolgáltatónál csak az engedélyezett szoftver fusson a felhőben?

**Comment by renced42** [comment]
@MicraMetripond 

Mint olyan, hogy felhő szolgáltató nincs, felhő alapú megoldás van a tervezetben, mely alapján a felhőalapú e-pénztárgép, olyan megoldás, amelyben a forgalmazó által biztosított alkalmazás (továbbiakban: alkalmazás) ténylegesen használt példányának, illetve a kezelő eszközéhez csatlakozó perifériák összessége látja el az e-pénztárgép funkcióit és teljesíti a rendeletben meghatározott követelményeket.
